description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
+- feedback from the teacher or peers.
+
+The presence and length of each of these activities during a practical session is influenced by the preference of the teacher.
+As such, the educational materials used as support for the practical sessions should offer the flexibility of being tailored depending on the teacher's needs.
+At the same time, they should be usable in the absence of a teacher as standalone learning materials for self-taught individuals that want to learn by themselves.
+
+To meet these constraints, educational materials for practical sessions should abide to the following format.
+A practical sessions is comprised of a series of concepts that the learner must assimilate.
+A concept must contain:
+
+- a brief theoretical description (which may contain links to more broader explanations) by using reading and media materials.
+- one or more practical exemplifications of the described concept by using demos.
+- one or more tutorials where the learner makes the first steps in applying the described concept.
+- one or more quizzes regarding some aspects of what has been presented so far.
+- one or more drills where the learner applies the presented concept.
+
+There is no enforcement on the order in which the above content types should be present in the concept layout, because that depends on the nature of the presented concept.
+However, we do recommend that the content types are mingled toghether so that the learner's attention is kept alive.
+For example, having a large theoretical description or a large demo may bore some students.
+To avoid that, quizzes, drills and tutorials may be sprinkled throught the concept layout to assure more engagement.
+
+Ideally, each practical activity (quiz, tutorial, drill) would be automatically evaluated and feedback would be instantly generated, if possible.
+For example, in computer programming courses, where drills and tutorials are typically comprised of writting programs, a checker should also accompany them.
+Quizzes, if employed, are automatically checked for correctness.
+Therefore, for humanitarian fields of study such as history or philosophy, we recommend using quizzes instead of drills as a form of practical activity.
+
+The number of concepts and the ordering in which the concepts are presented in a practical session is highly dependent on the nature of the field of study and the preference of the teacher.
+As a consequence, we do not have any recommendations regarding this aspect.
+However, if the content creator develops materials that have dependencies between them, this should be specified.
+
+Some teachers use slides to perform the brief theoretical description.
+Since this is a subjective choice, as content creators we do not provide slides for practical sessions.
+Rather, we create all of the concepts and it is up to the user to choose between them and create additional materials, such as slides.
```

Comment:
This is imprecise. We do provide slides, and they can be used during practical activities. There is no reason the `slides` content type should only be used for lectures.